src/Utils/filterSelectQueryConfig.ts

```diff
@@ -0,0 +1,12 @@
+import {QueryTypes, QueryOptionsWithType} from "sequelize";
+
+export function filtersSelectQuery(nodeId: number, filter: string): QueryOptionsWithType<QueryTypes.SELECT> {
+    return {
+        replacements: {
+            filter: `1 ${filter}`,
+            nodeId: nodeId,
+            period: filter == "day" ? "hour" : "day"
```

Comment:
oh, I was thinking only for the `period` but it's ok I guess like this as well :)